README.md

```diff
@@ -186,493 +186,14 @@ FreeBSD:
 Role Variables
 --------------
 
-This role has multiple variables. The defaults for all these variables are the following:
+This role has multiple variables. The descriptions and defaults for all these variables
+are in the directory defaults/main in the following files:
+
+- [main.yml - Common variables](defaults/main/main.yml)
```

Comment:
You could also point to the GitHub file location directly
```suggestion
- [main.yml: Common variables](https://github.com/nginxinc/ansible-role-nginx/blob/master/defaults/main.yml)
```
This will not work until the PR is merged but it should work after that.